_observing-your-data/event-analytics.md

```diff
@@ -24,33 +24,61 @@ For example, the following PPL query returns a count of how many host addresses
 source = opensearch_dashboards_sample_data_logs | fields host | stats count()
 ```
 
-By default, Dashboards shows results from the last 15 minutes of your data. To see data from a different timeframe, use the date and time selector.
+By default, Dashboards shows results from the last 15 minutes of your data. To see data from a different time frame, use the date and time selector.
 
 For more information about building PPL queries, see [Piped Processing Language]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index).
 
-## Save a visualization
+## Saving a visualization
 
 After Dashboards generates a visualization, you must save it if you want to return to it at a later time or if you want to add it to an [operational panel]({{site.url}}{{site.baseurl}}/observing-your-data/operational-panels).
 
 To save a visualization, expand the save dropdown menu next to **Refresh**, enter a name for your visualization, then choose **Save**. You can reopen any saved visualizations on the event analytics page.
 
-## View logs
+## Adding event analytics visualizations to dashboards
+
+Presenting your visualizations on a dashboard, instead of the event analytics page, makes it easier for users with varying levels of technical knowledge to understand and interpret the data at a glance. To create a new visualization and add it to a dashboard, follow these steps:
+
+1. On the main menu, select **Observability** > **Event analytics**.
+2. On the Event analytics window, enter the index source in the PPL query field, for example, `source = opensearch_dashboards_sample_data_logs | stats max(bytes), avg(bytes) by host`.
```

Comment:
*in* the event analytics window? Should "PPL query" be bold if it's a UI element? Should we specify that you have to enter it in the PPL language or is it obvious because of the field name?